x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/utils.ts

```diff
@@ -11,12 +11,12 @@ import { VectorScoreBase, Vector } from './types';
 
 export const vulnerabilitiesColumns = {
```

Comment:
nit and unrelated to this specific PR, but I think the `vulnerabilitiesColumns`, `defaultColumnProps`, `getVulnerabilitiesColumnsGrid` are not very fitting in a `utils` file, I wouldn't expect to find them here if I was looking. i think that them being used only once in the table component itself is also justifying for that. maybe we can create a new file `vulnerabilities_table_columns` and export the columns and such from there? (including the cell renderer function)

Reply:
makes sense, thanks